src/implementation/Server/DaprServer.ts

```diff
@@ -35,19 +36,19 @@ export default class DaprServer {
   readonly client: DaprClient;
 
   constructor(
-    serverHost = "127.0.0.1"
-    , serverPort: string = process.env.DAPR_SERVER_PORT || "50050"
-    , daprHost = "127.0.0.1"
-    , daprPort = "50051"
+    serverHost?: string
+    , serverPort?: string
+    , daprHost?: string
+    , daprPort?: string
     , communicationProtocol: CommunicationProtocolEnum = CommunicationProtocolEnum.HTTP
     , clientOptions: DaprClientOptions = {
       isKeepAlive: true
     }
   ) {
-    this.serverHost = serverHost;
-    this.serverPort = serverPort;
-    this.daprHost = daprHost;
-    this.daprPort = daprPort;
+    this.serverHost = serverHost ?? Settings.getDefaultHost();
+    this.serverPort = serverPort ?? Settings.getDefaultAppPort(communicationProtocol);
```

Comment:
Also here, should we check for `DAPR_SERVER_PORT`? I don't see it documented anywhere, but we do set that environment variable only in `DaprServer`.

Reply:
No I wouldn't set it. Let's stick to the Dapr documented one, let's work towards deprecating the DAPR_SERVER_PORT in the future as well?

Reply:
Cool, we don't have `DAPR_SERVER_PORT` in the new code. Resolving this.